Assets/HoloToolkit-Examples/GameControllerManipulator/Example.unity

```diff
@@ -0,0 +1,518 @@
+%YAML 1.1
```

Comment:
I agree but I was following the example set by ColorPicker: HoloToolkit-Examples/<Examplename>/Example.unity. Other examples don't have proper names under their folders either. I'll change mine as suggested.